bin/burden

```diff
@@ -2813,95 +2813,7 @@ verify_test_configuration()
-			if [[ $location == "https://github.com"* ]]; then
-				repo=`echo $location | cut -d'/' -f 1-5`
-				git -c 'versionsort.suffix=-' ls-remote --exit-code --refs --sort='version:refname' --tags $repo | rev | cut -d'/' -f1 | rev >  git_vers
-				#
-				# Make sure the tag we want is present
-				#
-				version_fail=""
-				grep ${using_version} git_vers > /dev/null
-				if [ $? -ne 0 ]; then
-					version_fail="$repo, version $using_version is not defined."
-					bail_out=1
-				fi
-				latest=`tail -1 git_vers`
-				if [[ $using_version != $latest ]]; then
-					version_ok="no"
-				else
-					version_ok="yes"
-				fi
-				if [[ $gl_update_test_versions -eq 0 ]]; then
-					printf '%32s%10s%10s%6s\n' $test_repo $using_version $latest $version_ok
-					if [[ $version_fail != "" ]]; then
-						echo Failure: $version_fail
-					fi
-				else
-					if [[ $version_ok == "no" ]]; then
-						#
-						# We have to walk the file instead of using a series of sed pipes.
-						# The reasons for this are:
-						#  1) We can not simply update the version number through the whole
-						#     file, we might change something we should not.  For example,
-						#     if an option passed to the test happens to match the current
-						#     version number in the file.
-						#       Example:
-						#         exec_dir: "test-wrapper-1.0/test"
-						#         repo_file: "v1.0.zip"
-						#         test_specific: "--option burden_version_1.0
-						#  2) If we do a full sed replacement, if there are extra spaces
-						#     in the line, we will not change it.
-						#  3) We could pull the required string via grep, and build the new
-						#     one.  However that will require reading the file 3 times (2
-						#     greps, and then one update), versus the one read if we simply
-						#     walk the file, and look for the line to replace.
-						#
-						echo Updating version for $test_repo from $using_version to $latest
-						while IFS= read -r replace_line
-						do
-							if [[ $replace_line == "repo_file:"* ]]; then
-								echo $replace_line | sed "s/$using_version/$latest/g" >> ${test_repo}.new
-								continue
-							fi
-							if [[ $replace_line == "exec_dir:"* ]]; then
-								replace_with=`echo $latest | sed "s/^v/-/g"`
-								replacing=`echo $using_version | sed "s/^v/-/g"`
-								echo $replace_line | sed "s/$replacing/$replace_with/g" >> ${test_repo}.new
-								continue
-							fi
-							echo $replace_line >> ${test_repo}.new
-						done < "${test_repo}"
-						mv $test_repo ${test_repo}.orig
-						mv ${test_repo}.new ${test_repo}
-					fi
-				fi
-				rm -rf $temp_dir 
-			else
-				echo $test_repo, Only github is supported.
-			fi
-		done
-		popd > /dev/null
-		if [ $gl_update_test_versions -eq 1 ]; then
-			cleanup_and_exit "Update of test versions completed" 0
-		fi
+		source $UTILS_DIR/test_versions_ops --update_test_versions $gl_update_test_versions --test_version_check $gl_test_version_check --config_dir config
```

Comment:
Is `source` needed here?

Reply:
Yes, we can exit out in the script.  We can move that logic back in, but it will be duplicated in both locations.